torch/csrc/inductor/resize_storage_bytes.cpp

```diff
@@ -38,8 +38,14 @@ static void resize_storage_bytes__functionalize(
     op.call(variable, new_size);
     return;
   }
+  // Don't functionalize, call the mutable op on the inner tensor.
   auto functional_impl =
       at::functionalization::impl::unsafeGetFunctionalWrapper(variable);
+  {
+    at::AutoDispatchSkipFunctionalize guard;
+    op.call(functional_impl->value(), new_size);
+    return;
+  }
   // Sync pending mutations before running the resize_()
```

Comment:
You can kill the 3 lines below this, now that we're making the functionalization rule for `resize_` "no-op"